src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt

```diff
@@ -57,6 +58,34 @@ class ContextFilesPanel(
     add(accordionSection, BorderLayout.CENTER)
   }
 
+  private fun deriveAccordionTitle(contextFileFiles: List<ContextFileFile>): String {
+    val uniqueFiles = mutableSetOf<String>()
+
+    val filteredFiles =
+        contextFileFiles.filter { file ->
+          if (uniqueFiles.contains(file.uri.toString())) {
+            false
+          } else {
+            uniqueFiles.add(file.uri.toString())
+            true
+          }
+        }
+
+    val prefix = "✨ Context: "
+    val lineCount = contextFileFiles.sumOf { it.range?.delta() ?: 0 }
```

Comment:
Readability of this line is problematic at first glance. What's the meaning of "delta"? Proposal: `delta` -> `numberOfLines`?

Reply:
Or even just `length`, as `range.length` will be quite self explaining.

Reply:
good point!